host/4/bullseye/amd64/java/java8/java8-composite.template

```diff
@@ -1,7 +1,7 @@
 ARG JAVA_VERSION=8u302b08
 ARG JDK_NAME=jdk8u302-b08
 ARG JAVA_HOME=/usr/lib/jvm/adoptium-8-x64
-FROM mcr.microsoft.com/dotnet/runtime-deps:6.0.0
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
```

Comment:
@balag0 / @CooperLink - should all docker file be using `mcr.microsoft.com/dotnet/runtime-deps:6.0` instead of `FROM mcr.microsoft.com/dotnet/runtime-deps:6.0` ?

Reply:
@CooperLink - merging this PR to ensure next release includes the SDK update